modules/introduction/pages/whats-new.adoc

```diff
@@ -1,44 +1,162 @@
+
+Couchbase transactions are architected to provide ACID guarantees at scale while ensuring there are no single points of failure or performance bottlenecks. 
+
+For more information, see xref:learn:data/distributed-acid-transactions.adoc[Distributed ACID Transactions].
+
+==== SDK 3.0 Alpha Releases
+
+Couchbase SDKs are bringing in major new features and updating interfaces to the latest for the platform. They support many new features available in this Server release while ensuring compatibility with older releases. 
+
+==== Query Enhancements
+
+This release packs numerous additions to the Query service and N1QL to provide a comprehensive query development toolkit for building applications. 
+
+===== Window Functions
+
+RDBMS ANSI standard equivalent window functions are introduced in Couchbase to simplify complex queries like financial and statistical reporting to intuitive N1QL constructs.
+
+Window functions are used to compute an aggregate or cumulative value, based on a group of objects. The objects are not grouped into a single output object — each object remains separate in the query output.
+
+For more information, see xref:n1ql:n1ql-language-reference/windowfun.adoc[Window Functions].
+
+===== Common Table Expressions (CTE)
+
+Couchbase CTE is yet another ANSI CTE equivalent to enhance the manageability of complex queries where the developer can isolate the SQL statement into a temporary named result set which can be referenced as a source table in the context of a larger query. 
+
+For more information, see xref:n1ql:n1ql-language-reference/with.adoc[WITH Clause].
+
+===== Query and Search Integration
+
+Couchbase’s powerful full-text search capability is now fully integrated with N1QL. The feature extends N1QL search functionality with fuzzy and non-exact text searches across multiple fields and any order. By enabling developers to use full-text search queries directly within a N1QL query, it eliminates the need to write complex code to process and combine the results from separate SQL and search queries.
+
+For more information, see xref:n1ql:n1ql-language-reference/searchfun.adoc[Search Functions].
+
+===== Cost Based Optimizer (Developer Preview)
+
+Couchbase is delivering the well-known cost-based optimizer, deriving access patterns based on statistical data for query tuning to the NoSQL world. This feature aids developers by eliminating the time spent on tweaking a query using a rule-based optimizer to choose the most optimal execution plan. 
```

Comment:
This para has me scratching my head a bit. What is well-known? Is this something readers are familiar with? The first sentence is a bit hard to read.

Reply:
Cost-based optimizer is a well-known concept in the RDBMS world.